index.js

```diff
@@ -4,7 +4,7 @@ const accepts = require('accepts')
 
 const fp = require('fastify-plugin')
 
-const acceptsObjectSymbol = Symbol('acceptsObject')
+const acceptsObjectSymbol = Symbol.for('acceptsObject')
```

Comment:
Keep it `Symbol('acceptsObject')`. It is a internal value for this plugin and shouldn't be modify by others.

Reply:
```suggestion
const acceptsObjectSymbol = Symbol('acceptsObject')
```